scrap-fundamentals.js

```diff
@@ -0,0 +1,94 @@
+const axios = require('axios');
```

Comment:
This is the main file. Importing/Parsing/Generating md files. All happening in this file.